ui/pages/confirm-approve/confirm-approve-content/confirm-approve-content.component.js

```diff
@@ -178,7 +181,60 @@ export default class ConfirmApproveContent extends Component {
     );
   }
 
-  renderPermissionContent() {
+  renderERC721OrERC1155PermissionContent() {
+    const { t } = this.context;
+    const { origin, toAddress, isContract, assetName, tokenId } = this.props;
+
+    const displayedAddress = isContract
+      ? `${t('contract')} (${addressSummary(toAddress)})`
+      : addressSummary(toAddress);
+    return (
+      <div className="flex-column">
+        <div className="confirm-approve-content__small-text">
+          {t('accessAndSpendNoticeNFT', [origin])}
+        </div>
+        <div className="flex-row">
+          <div className="confirm-approve-content__label">
+            {t('approvedAsset')}:
+          </div>
+          <div className="confirm-approve-content__medium-text">
+            {`${assetName} #${tokenId}`}
+          </div>
+        </div>
+        <div className="flex-row">
+          <div className="confirm-approve-content__label">
+            {t('grantedToWithColon')}
+          </div>
+          <div className="confirm-approve-content__medium-text">
+            {`${displayedAddress}`}
```

Comment:
I don't think this needs to be in a string

Reply:
done here: https://github.com/MetaMask/metamask-extension/pull/13131/commits/8c1109875b2bd628c85dd5554e316bb2b53582a5